core/misc/ckeditor/CHANGES.md

```diff
@@ -1,2117 +1,1879 @@
-CKEditor 4 Changelog

```

Comment:
TBH, I don't think we're supposed to fix external sources like the ckeditor changelog. I will get overridden anyway with the next ckeditor update.